src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -832,6 +832,14 @@ GREENHOUSE_API_KEY = '{{ GREENHOUSE_API_KEY }}'
 LOCAL_LOGGING_CONFIG = json.loads('{{ localsettings.get('LOCAL_LOGGING_CONFIG', {})|to_json }}')
 LOCAL_MIDDLEWARE = {{ localsettings.get('LOCAL_MIDDLEWARE', [])|to_json }}
 
+{% if 'SECURE_HSTS_SECONDS' in localsettings %}
+SECURE_CONTENT_TYPE_NOSNIFF = {{ localsettings.get('SECURE_CONTENT_TYPE_NOSNIFF', True) }}
+SECURE_BROWSER_XSS_FILTER = {{ localsettings.get('SECURE_BROWSER_XSS_FILTER', True) }}
+SECURE_HSTS_SECONDS = {{ localsettings.get('SECURE_HSTS_SECONDS', 0) }}
```

Comment:
This should be set in nginx directly: https://github.com/dimagi/commcare-cloud/blob/master/environments/production/proxy.yml#L10